src/Azs.Compute.Admin/readme.md

```diff
@@ -55,7 +55,7 @@ metadata:
 
 ### PSD1 metadata changes
 subject-prefix: ''
-module-version: 1.1.0
+module-version: 1.1.1
 service-name: ComputeAdmin
```

Comment:
Please update the change log as well https://github.com/Azure/azurestack-powershell/blob/dev/src/changelog.md
Code review checklist is at https://github.com/Azure/azurestack-powershell/wiki/Code-Review-Check-List